src/components/metadata/styles/metadata.scss

```diff
@@ -4,24 +4,33 @@
 .kui-theme--light {
   --color-metadata-bg: #{$color-bg-light-3};
   --color-metadata-kind-token-bg: #{$color-bg-light-2};
+  --color-plot-bg: #{$color-bg-light-5};
+  --color-button-plot: #{$color-button};
```

Comment:
I have been getting an 'undefined variable' error for the `color-button` variable on running the app from this branch, and indeed I do notice that `color-button` is not defined in `_variables.scss` or anywhere in the app- I wonder if there are any further new changes to the variables scss file that hasn't been pushed?

Reply:
thanks for highlighting. this is fixed